tracking/ZR_tracking.py

```
@@ -0,0 +1,567 @@
+                    DatabaseUtils.write_table(self.engine, log_df, self.change_log_table_name, index=False)
+                    
+                    # Print summary of changes
+                    print("\nChanges by Zone:")
+                    for zr in log_df['ZR'].unique():
+                        zr_changes = log_df[log_df['ZR'] == zr]
+                        print(f"\nZone: {zr}")
+                        
+                        #for every row corresponding to the particular zone, print changes that apply and date of change
+                        for _, change in zr_changes.iterrows():
+                            if change['field_changed'] == 'habilitada':
+                                print(f"  - New zone added")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)